src/menu/views/browser.c

```diff
@@ -343,6 +343,7 @@ static component_context_menu_t settings_context_menu = {
     .list = {
         { .text = "Menu settings", .action = set_menu_next_mode, .arg = (void *) (MENU_MODE_SETTINGS_EDITOR) },
         { .text = "Time (RTC) settings", .action = set_menu_next_mode, .arg = (void *) (MENU_MODE_RTC) },
+        { .text = "Controller Pak settings", .action = set_menu_next_mode, .arg = (void *) (MENU_MODE_CONTROLLER_PAKFS) },
```

Comment:
Move this above "Menu Settings", to be the first option (for most users, I think that is what would be prefered).
Also, Consider using the title "Controller Pak manager", as seems more informative.